app/controller/command/commands/team.py

```diff
@@ -385,8 +386,9 @@ def add_helper(self, param_list, user_id) -> ResponseTuple:
         """
         try:
             command_user = self.facade.retrieve(User, user_id)
+            command_team = param_list['team_name']
             teams = self.facade.query(Team, [('github_team_name',
-                                              param_list['team_name'])])
+                                              command_team)])
             if len(teams) != 1:
                 return self.lookup_error, 200
             team = teams[0]
```

Comment:
Check `db.utils.get_team_by_name()` that does the same thing with error handling.

Reply:
285f80f41aeec17c3a35c2002d18c87b323d73cb